dist/scss/variables.scss

```diff
@@ -111,6 +111,6 @@ $navigationHoverColor: #191D22;
 // Overlay
 // -------------------------
 $overlayContainerWidthSmall: 580px;
-$overlayContainerWidthMedium: 700px;
+$overlayContainerWidthMedium: 860px;
```

Comment:
The breakpoints should be fixed now. As far as I know the breakpoints only affected the media-selection.